### ISSUE TITLE
bad manual release without builds

### ISSUE THREAD

**Issue by devopsmariocom** [wc-catalogue/blaze-elements#317]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => You can submit support request here, but we preffer to chat about it realtime at our [ngParty slack](https://ngparty.herokuapp.com/)
```

**Current behavior**
I have manualy released but forgot to run build before triggering release, so current releases are empty :(

**Expected behavior**
We have to add build step to manual release and trigger release of all packages using proper commit message

**Comment by devopsmariocom** [comment]
partially closed by #316 but seems only packages changes in git are released even that I have listed affects for all packages.

I will create another PR to release all packages specially blaze-elements

**Comment by devopsmariocom** [comment]
actually my previous assumption was wrong, for example badge was changed in git and mentioned in affects but not properly released see 

```
[32minfo[39m:    Executing getNpmConfig in packages/badge
[32minfo[39m:    Executing makeSrConfig in packages/badge
[32minfo[39m:    Executing pre in packages/badge
[32minfo[39m:    Anaylzed 0 / 27 commits to determine type null for @blaze-elements/badge
[31merror[39m:    message=There are no relevant changes, so no new version is released., code=ENOCHANGE
```

https://travis-ci.org/wc-catalogue/blaze-elements/builds/213749718 this might be caused by so many affected packages in single commit but this release hell doesn't seems to go anywhere...

I will try to do blaze-elements release to see if that works.
